numpy/core/src/umath/scalarmath.c.src

```diff
@@ -15,6 +15,7 @@
 #include "numpy/arrayobject.h"
 #include "numpy/ufuncobject.h"
 #include "numpy/arrayscalars.h"
+#include "scalartypes.h"
```

Comment:
Is this write place to add it? Any logical grouping of the headers done here? I noticed there are groups

Reply:
Honestly, the header includes are probably just a mess all over the place, and I am not sure of much logic... I would probably group it by where the folders (which may mean putting it by itself here). I doubt the rest follows any such logic though. I am willing to bet that most of the order is simply the historical order for when it was necessary to add them...

Reply:
Oh I see. Yeah, I'll make it separate.